tests/phpunit/tests/link/getPostCommentsFeedLink.php

```diff
@@ -80,8 +81,40 @@ public function test_attachment_pretty_link() {
 		$this->assertSame( $expected, $link );
 	}
 
-	public function test_attachment_no_name_pretty_link() {
+	public function test_attachment_pretty_link_attachment_pages_off() {
```

Comment:
`test_attachment_pretty_link_attachment_pages_off()` explains the circumstances, but not the expected behaviour.

While we're touching these tests and adding new ones, can we document the expected behaviour in the tests' docblocks?

:1234: Applies elsewhere in the PR.

Reply:
> `test_attachment_pretty_link_attachment_pages_off()` explains the circumstances, but not the expected behaviour.

@costdev Do you have a suggestion for an improved name for each of these? What would make it clearer for you?

Reply:
For example, maybe something like this?

- `test_attachment_pretty_link_with_attachment_pages_on_uses_slug`
- `test_attachment_pretty_link_with_attachment_pages_off_uses_hash`

Reply:
There are various edits to the comment above. Any of these may be suitable/preferred.